content/src/Server.ts

```diff
@@ -134,6 +134,7 @@ export class Server {
     await this.snapshotManager.start();
     await this.synchronizationManager.start();
     await this.garbageCollectionManager.start();
+    await this.service.start();
```

Comment:
Let's move this before 
```this.httpServer = this.app.listen(this.port);```

That way, the deployments count is made before HTTP requests can be made, and therefore before new deployments can be made